src/Signup.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+
+const Signup = (): JSX.Element => {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [confirm, setConfirm] = useState("");  
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    // Do something with name, email, password, and confirmed password.
+    /* console.log(`name: `, name);
+     * console.log(`email: `, email);
+     * console.log(`password: `, password);
+     * console.log(`confirm: `, confirm);     */
+  };
+
+  return (
+    <>
+	<h2>Sign Up</h2>
+	<form onSubmit={(e) => handleSubmit(e)}>
+            <label>
+		Name:
+		<input
+		  type="text"
```

Comment:
Would be nice to have a [placeholder](https://www.w3schools.com/tags/att_input_placeholder.asp).